### DISCUSSION TITLE
Possibility of editing tags manually in mympd?

### DISCUSSION THREAD

**Discussion by bartfer** [jcorporation/myMPD#989] [Ideas]
My goal is to somehow add various free-form comments to songs (for example describing it in my own words or connecting it to a specific experience so that I can search for it easier later).
I could do this from linux terminal by editing tags for the files (at least I think, but .opus and .mp3 metadata structure seems a bit different but my guess is it can be done in both).

(btw my first try was this: https://beets.io/blog/flexattr.html which works great but there is basically 0 possibility that I could execute beet commands from mympd +? and also even if I could I'd not be able to search in mympd, so overall it'd be better to include these comments directly in the song files in my opinion)

Would it be possible to somehow edit tags in mympd? I see the comments tab when checking song details, would be great if we could edit manually there or maybe add new field (if file format allows us?). I mean these tags are probably predefined for a specific format but there is a very promising "comment" tag I've seen. Unfortunately I couldn't really find proper information about these yet.

Any ideas?

Sorry if I made some dumb assumptions, I'm still learning this topic. (BTW, best mpd client I've found. :) )

**Comment by jcorporation** [comment]
MPD itself and therefore myMPD has no tag editing functionality. MPD supports stickers for freeform labels for songs. Unfortunately you can not search for stickers like it is possible for tag values.

Adding freeform stickers to songs and searching could be an enhancement for myMPD.

Stickers are saved in a sqlite database by MPD. There are no commands to search for tags and stickers simultaneously. You can search for stickers or for tags. This is a MPD limitation.

**Reply by bartfer** [reply]
I understand. Thank you for the quick response!
